### PR TITLE
Add Vanuatu holidays

### PR DESCRIPTION
**Pull request by Strategos1**
## Proposed change

Your PR description goes here.

## Type of change

- [x] New country/market holidays support (thank you!)
- [ ] Supported country/market holidays update (calendar discrepancy fix, localization)
- [ ] Existing code/documentation/test/process quality improvement (best practice, cleanup, refactoring, optimization)
- [ ] Dependency update (version deprecation/upgrade)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (a code change causing existing functionality to break)
- [ ] New feature (new python-holidays functionality in general)

## Checklist

- [x] I've followed the [contributing guidelines][contributing-guidelines]
- [x] This PR is filed against `beta` branch of the repository
- [x] This PR doesn't contain any merge conflicts and has clean commit history
- [x] The code style looks good: `make pre-commit`
- [x] All tests pass locally: `make test`, `make tox` (we strongly encourage adding tests to your code)
- [x] The related [documentation][docs] has been added/updated (check off the box for free if no updates is required)

[contributing-guidelines]: https://github.com/vacanza/python-holidays/blob/beta/CONTRIBUTING.rst
[docs]: https://github.com/vacanza/python-holidays/tree/beta/docs/source

### CODE REVIEW

File: tests/countries/test_vanuatu.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    def test_no_holidays(self):
+        self.assertNoHolidays(Vanuatu(years=1980))
+
+    def test_fater_lina_day(self):
+        name = "Father Lini Day"
+        self.assertHolidayName(name, (f"{year}-2-21" for year in range(1991, 2050)))
+        self.assertNoHolidayName(name, Vanuatu(years=range(1981, 1991)))
+        self.assertNoHoliday(
+            f"{year}-2-21" for year in set(range(1981, 1991)).difference({1982, 1984, 2085})
+        )
```
Comment (KJhellico) [inline tests/countries/test_vanuatu.py:33]:
```suggestion
    def test_fater_lini_day(self):
        name = "Father Lini Day"
        self.assertHolidayName(name, (f"{year}-02-21" for year in range(1991, 2050)))
        self.assertNoHolidayName(name, Vanuatu(years=range(1981, 1991)))
        self.assertNoHoliday(f"{year}-02-21" for year in range(1981, 1991))
```
I don't see any reasons for exceptions for 1982, 1984.

**Review by KJhellico — COMMENTED** [review]
@Strategos1 , thank you for adding new countries to Python Holidays! Please look at suggestions below.

File: tests/countries/test_vanuatu.py (hunk 1 above)
Comment (Strategos1) [reply tests/countries/test_vanuatu.py]: I copied a code from another country and forgot to change it. Sorry

File: tests/countries/test_vanuatu.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    def setUpClass(cls):
+        super().setUpClass(Vanuatu)
+
+    def test_country_aliases(self):
+        self.assertCountryAliases(Vanuatu, VU, VTU)
+
+    def test_no_holidays(self):
+        self.assertNoHolidays(Vanuatu(years=1980))
+
+    def test_fater_lini_day(self):
```
Comment (arkid15r) [inline tests/countries/test_vanuatu.py:27]:
```suggestion
    def test_father_lini_day(self):
```

**Review by arkid15r — COMMENTED** [review]
lgtm, the code is clean and tested. If no comments regarding the calendar correctness I believe it's ready for merging into beta.

Thank you @Strategos1 for such a great contribution! Great work!

**Review by KJhellico — COMMENTED** [review]
LGTM 🚀

**Comment by arkid15r** [comment]
Merged into beta, thanks for extending the list of the supported countries 👍

**Pull request merged**